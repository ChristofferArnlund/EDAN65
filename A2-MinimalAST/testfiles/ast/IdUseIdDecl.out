Program
  List
    Func ID="a"
      List
      List
        IdDecl ID="x"
        IdDecl ID="y"
        IdUse ID="x"
          Numeral NUMERAL="5"
        IdUse ID="y"
          Name ID="x"
