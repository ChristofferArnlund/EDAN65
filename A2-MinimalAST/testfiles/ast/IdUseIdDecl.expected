Program
  List
    Func ID="a"
      List
      List
        IdDecl ID="x"
          Opt
        IdDecl ID="y"
          Opt
        IdUse ID="x"
          Numeral NUMERAL="5"
        IdUse ID="y"
          Name ID="x"
